code/Controllers/SilverStripeNavigatorItem.php

```diff
@@ -29,6 +33,12 @@ abstract class SilverStripeNavigatorItem extends ViewableData
      */
     public function __construct(CMSPreviewable $record)
     {
+        Deprecation::notice(
+            '4.13.0',
+            'Use SilverStripe\Admin\Navigator\SilverStripeNavigatorItem instead',
+            Deprecation::SCOPE_CLASS
+        );
```

Comment:
```suggestion
        Deprecation::withNoReplacement(function () {
            switch (static::class) {
                case 'SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_Unversioned':
                case 'SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink':
                case 'SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink':
                case 'SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink':
                    // These will have their own appropriate deprecation notices
                    break;
                default:
                    Deprecation::notice(
                        '4.13.0',
                        'Will be renamed SilverStripe\Admin\Navigator\SilverStripeNavigatorItem',
                        Deprecation::SCOPE_CLASS
                    );
            }
        });
```
Each class should have it's _own_ appropriate deprecation notice, stating the correct new FQCN.

Reply:
DONE